man/systemd-journald.service.xml

```diff
@@ -276,12 +321,13 @@ systemd-tmpfiles --create --prefix /var/log/journal</programlisting>
         <term><filename>/run/systemd/journal/socket</filename></term>
         <term><filename>/run/systemd/journal/stdout</filename></term>
 
-        <listitem><para>Sockets and other paths that
-        <command>systemd-journald</command> will listen on that are
-        visible in the file system. In addition to these, journald can
-        listen for audit events using netlink.</para></listitem>
+        <listitem><para>Sockets and other paths that <command>systemd-journald</command> will listen on that
```

Comment:
This does not appear to be a complete sentence.

Reply:
will fix